.infrastructure/i18n/extract-server-msgs.py

```diff
@@ -0,0 +1,14 @@
+import os
```

Comment:
Python scripts which are basically just a list of `os.system` calls, I prefer to have as bash scripts instead. Python isn't really verbose or clear when doing os.system per default, making it cumbersome to debug. I'd vote for replacing this file with a bash scripts (`.sh`)...